RELEASE.md

```diff
@@ -20,6 +20,9 @@
 * Added documentation to clarify execution order of hooks.
 ## Breaking changes to the API
 ## Upcoming deprecations for Kedro 0.19.0
+* All dataset classes will be removed from the core Kedro repository (`kedro.extras.datasets`). Install and import them from the [`kedro-datasets`](https://github.com/kedro-org/kedro-plugins/tree/main/kedro-datasets) package instead.
```

Comment:
Was this a new deprecation? This was already noted in 0.18.8 (see this same document, far below)

Reply:
Ah I missed that.. so technically it's not new, but in `0.18.8` it fell under documentation, and I also feel that this is probably significant enough to mention again. WDYT?

Reply:
Doesn't hurt to emphasize it there again! 👍🏽